python/mleap/sklearn/preprocessing/data.py

```diff
@@ -532,11 +533,19 @@ def __init__(self):
 
     def serialize_to_bundle(self, transformer, path, model_name):
 
-        # compile tuples of model attributes to serialize
+        if transformer.strategy == 'most_frequent' or transformer.strategy == 'constant':
+            raise ValueError(f"Imputer strategy `{transformer.strategy}` is not supported by MLeap")
+        if isinstance(transformer.missing_values, str):
+            raise ValueError("Imputer missing values of type `str` are not supported by MLeap")
+        if transformer.add_indicator:
+            raise ValueError("Imputer parameter `add_indicator` is not supported by MLeap")
+        if len(transformer.statistics_.tolist()) != 1:
+            raise ValueError("MLeap Imputer only supports one feature at a time")
```

Comment:
It seems that on the mleap side you have to specify a single input and output column:
https://github.com/combust/mleap/blob/d276c1c800277a1edcf30dda6e8559c7413b21f8/mleap-runtime/src/test/scala/ml/combust/mleap/runtime/transformer/feature/ImputerSpec.scala#L14